lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -698,7 +711,7 @@ def update_task_progress():
     report_store = ReportStore.from_config()
```

Comment:
I'm also hesitant about factoring the ora2 code because it looks like the right way to track progress is with this class: https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor_task/tasks_helper.py#L169